README.md

```diff
@@ -184,6 +184,9 @@ When a PR is created and the performance tests are needed, the following comment
 
 It will launch the `kroxylicious-performance-tests-pr` build, that will insert a comment with a summary into the PR comparing the results with the previous execution.
 
+> **Warning**\
+> The comparison with the previous version has to be checked manually as currently there is no pass/fail criteria in the pipeline
```

Comment:
I don't really think we need to mention the behaviour of the old version.  That irrelevant detail.

Reply:
removed